proto/brpb.proto

```diff
@@ -410,6 +427,26 @@ message BackupResponse {
     kvrpcpb.APIVersion api_version = 5;
 }
 
+message CleanSSTFilesRequest { 
+}
+
+message CleanSSTFilesResponse {
+    Error error = 1;
+    bool cleaned = 2;
+}
+
+message CollectSSTFilesRequest { 
+    // whether generate SST File map and save it to the storage.
+    bool save_to_storage = 1;
```

Comment:
I think we don't need to do it here
We'd better store it into `checkpoint` or `data-dir` automatically. WDYT?

Reply:
So if `save_to_stroage == false`, what should happen? Generate only a RocksDB checkpoint (with out region map) or trivially do nothing?

Reply:
if save_to_stroage == false. then we don't need to save some state like (key range -> SST files) to the storage.
And it may be used for optimization.